### DISCUSSION TITLE
Access to Project Template

### DISCUSSION THREAD

**Discussion by jakegros** [USU-ITLS-x265-23-summer/class-repo#178] [Q&A]
I am assuming others have been able to access the [Project Template](https://github.com/USU-ITLS-x265-23-summer/project).  Since I am unable to, will someone share the project template with me, please? 🙏

**Comment by bst3ck** [comment]
I am just seeing your note. Did you ever get access to the project template? If not, I may be able to help.

**Reply by jakegros** [reply]
Hey, I did not.  I'm not able to view yours either. I'm guessing there is some kind of permission issue with the root, which is weird since I can access everything else in the course, it seems.

**Reply by jakegros** [reply]
Actually, I am able to access your version of it in the repos. Now I've confirmed what was included in the template and instructions. It looks like just a basic file setup to begin with. I've copied that. It will just have to be in my own repo, then shared with everyone, I suppose. 
Thank you.
